package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "smarthouse-frontend",
+  "name": "smart-house-frontend",
```

Comment:
All dependencies  alredy moved to dev.  - With name seems that we will nead to follow same format over projects.